### ISSUE TITLE
httpSink Sink Depricated

### ISSUE THREAD

**Issue by BebeSparkelSparkel** [snoyberg/http-client#411]
https://hackage.haskell.org/package/conduit/docs/Data-Conduit.html#t:Sink

Deprecated: Use ConduitT directly

**Comment by snoyberg** [comment]
Please clarify what this means

**Comment by snoyberg** [comment]
No, it's name includes Sink. Unless I'm misreading everything, it does not use the deprecated identifier. Can you please look again?
